modules/ob-release-notes-1-2.adoc

```diff
@@ -5,20 +5,69 @@
 
 [id="new-features-1-2_{context}"]
 == New features
 
-The following sections highlight what is new in {builds-shortname} 1.2.
+The following section highlights what is new in {builds-shortname} 1.2.
 
 [id="builds-new-features-1-1_{context}"]
-=== Builds
+=== {builds-shortname}
 
+* With this update, you can use the {builds-title} on the {ocp-product-title} cluster that runs on ARM, IBM Power, and IBM Z worker hardware. The components to build images with Shipwright and Shared Resource CSI Driver are available.
+
+* With this update, you can deploy {builds-shortname} on the clusters that are configured to run behind a network proxy.
+
+* With this update, you can deploy {builds-shortname} on the clusters that run in a disconnected network environment.
+
+* With this update, you can use the `buildah` and `source-to-image` build strategies to mount a Red{nbsp}Hat subscription entitlement key from any supported volume source. This is useful when you need builds to install Red{nbsp}Hat subscription content by using `yum` or `dnf` commands.
+
+* With this update, you can create a Shipwright `Build` object by using the `buildah` or `source-to-image` build strategies from the examples provided in the *Developer* perspective of the {ocp-product-title} web console. You can change the parameters in these examples to create custom `Build` objects.
+
+* With this update, you can download the binaries for `shp` CLI from the *Developer* perspective of the {ocp-product-title} web console. It is available for the following operating systems and architectures:
+
+[options="header"]
+|===
+| Operating system | Architecture
+| Linux  | amd64
+| Windows | amd64
+| Darwin | amd64 (macOS Intel)
+|===
+
+[id="fixed-issues-1-2_{context}"]
+== Fixed issues
+
+The following section highlights fixed issues in {builds-shortname} 1.2.
+
+* Before this update, the Shared Resource CSI Driver was unable to mount `SharedSecret` and `SharedConfigMap` resources due to missing OpenShift RBAC permissions. With this update, the global permissions are granted to the Shared Resources CSI driver. Now, the Shared Resource CSI Driver can access `SharedSecret` and `SharedConfigMap` resources and perform the required permission checks without any additional action from the cluster administrators.
 
 [id="known-issues-1-2_{context}"]
 == Known issues
 
 The following section highlights known issues in {builds-shortname} 1.2.
 
+* The default build strategies fail on clusters with ARM, IBM Power, or IBM Z worker node CPU architectures. 
++
+As a workaround, perform the following steps:
+
+.. Download the following multi variant of the default `ClusterBuildStrategy` resources to your cluster:
+** `buildah-multi`: link:https://issues.redhat.com/secure/attachment/13306657/13306657_buildah-multi.yaml[buildah-multi.yaml]
+** `source-to-image-multi`: link:https://issues.redhat.com/secure/attachment/13306658/13306658_source_to_image_multi.yaml[source_to_image_multi.yaml]
```

Comment:
The links are correct and publicly accessible, but please check if it is the preferred way to provide the files. The docs team will have a guideline for this.

Reply:
Hello, I am not aware of any rules regarding this. i will run this through peer review and see their opinion. If they do not know, I will find the answer before merging. Thanks!

Reply:
I am OK with the links if the downloaded files have been tested and passed by QE.

Reply:
Is _multi variant_ a known term? I am not seeing too much relevant by using web searches. There are some uses of multi-variant as an adjective (multi-variant testing, for example).

Reply:
Sorry I missed your comments! Not sure about how well the term is known, I will keep it for not to not postpone the release,but I will add the internal website note, thanks!